src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj

```diff
@@ -1001,6 +1001,9 @@
     <None Include="ConverterTestData\ClangAnalyzer\RealLog.xml">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="DirectProducerTestData\Roslyn\RoslynLog.sarif">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```

Comment:
Does this mean that every time I run BuildAndTest.cmd, it will take 3 minutes longer to run? I suggest that instead you generate a much smaller Roslyn log to add to the sarif-sdk.